tools/dockerfile/grpc_artifact_linux_x64/Dockerfile

```diff
@@ -58,6 +58,20 @@ RUN apt-get update && apt-get install -y \
   wget \
   zip && apt-get clean
 
+
+##################
```

Comment:
Yeah, maybe. @murgatroid99  didn't like the idea too much and for artifacts I'm fine both ways. We definitely need separate & lightweight docker image for testing the packages though (which is the case currently).

Also, this just got merged, so let's discuss this and address in a followup PR.